google-map-point.html

```diff
@@ -0,0 +1,58 @@
+<!-- Copyright (c) 2015 Google Inc. All rights reserved. -->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-apis/google-maps-api.html">
+
+<!--
+The `google-map-point` element represents a point on a map. It's used as a child of other
+google-map-* elements.
+
+<b>Example</b>:
+
+    <google-map-point latitude="37.77493" longitude="-122.41942"></google-map-point>
+-->
+<dom-module id="google-map-point">
+  <style>
+    :host {
+      display: none;
+    }
+  </style>
+</dom-module>
+
+<script>
+  Polymer({
+    is: 'google-map-point',
+
+    properties: {
+      /**
+       * The point's longitude coordinate.
+       */
+       longitude: {
+         type: Number,
+         value: null,
+         notify: true,
```

Comment:
I dont think these need to notify. Nothing inside this element is changing the lat/lng and I wouldnt expect anything to.